packages/xo-server-sdn-controller/src/index.js

```diff
@@ -807,6 +1224,24 @@ class SDNController extends EventEmitter {
 
   // ---------------------------------------------------------------------------
 
+  _getPoolNetwork(poolRef, crossPoolNetwork) {
+    const xapi = find(this._xapis, xapi => xapi.pool.$ref === poolRef)
+    const networkRef = find(crossPoolNetwork.networks, networkRef => {
+      try {
+        const network = xapi.getObjectByRef(networkRef)
+        return network.$pool.$ref === poolRef
+      } catch (error) {
+        return false
+      }
+    })
+
+    const poolNetwork = find(this._poolNetworks, {
+      pool: poolRef,
+      network: networkRef,
+    })
+    return poolNetwork
```

Comment:
Why don't you return the network object you found above? Or are they different objects?

Reply:
a `poolNetwork` is:
- a `poolRef`
- a `networkRef`
- a `hostRef` representing the center to the star topology of the network.